virtme/commands/configkernel.py

```diff
@@ -89,8 +89,9 @@ def arg_fail(message):
 
 def main():
     args = _ARGPARSER.parse_args()
-
-    if not os.path.isfile('scripts/kconfig/merge_config.sh'):
+    if os.path.islink('source'):
+        kernel_source_prefix = 'source/'
+    if not os.path.isfile(kernel_source_prefix + 'scripts/kconfig/merge_config.sh'):
```

Comment:
kernel_source_prefix is not guaranteed to be defined here.

Reply:
ah...let me guarantee that.